src/pages/Profile.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import Header from '../components/Header';
+import ProfileIconDump from '../assets/images/ProfileRed.svg';
+import '../styles/Profile.scss';
+import '../styles/MyBadge.scss';
+import ProfileButton from '../components/ProfileButton';
+import BadgeGroup from '../components/BadgeGroup';
+
```

Comment:
```suggestion
import Chevrion from '../components/Chevrion';

```
Chevrion component import